File: Mobile/Demol/src/entity/Trajet.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entity;

import java.util.Date;

/**
 *
 * @author dainer
 */
public class Trajet {
    private int id;
    private String destination;
    private String depart;
    private String datedepart;
    private String datepub;
    private float cout;
    private String description;
    private int nbrplacedispo;
    private boolean suivie;
    private Vehicule vehciule;
    private User user;

        public Trajet(String destination, String depart, String dateDepart, String datepub, float cout, String description, 
        int nombrePlaceDispo, boolean suivie) {
        this.destination = destination;
        this.depart = depart;
        this.datedepart = dateDepart;
        this.datepub=datepub;
        this.cout = cout;
        this.description = description;
        this.nbrplacedispo = nombrePlaceDispo;
        this.suivie = suivie;
    }    

    public Trajet()
    {
        
    }
    public int getId()
    {
        return this.id;
    }

    public void setId(int id)
    {
        this.id = id;
    }

    public String getDestination()
    {
        return this.destination;
    }

    public void setDestination(String destination)
    {
        this.destination = destination;
    }

    public String getDepart()
    {
        return this.depart;
    }

    public void setDepart(String depart)
    {
        this.depart = depart;
    }

    public String getDatedepart()
    {
        return this.datedepart;
    }

    public void setDatedepart(String datedepart)
    {
        this.datedepart = datedepart;
    }

    public float getCout()
    {
        return this.cout;
    }

    public void setCout(float cout)
    {
        this.cout = cout;
    }

    public String getDescription()
    {
        return this.description;
    }

    public void setDescription(String description)
    {
        this.description = description;
    }

    public int getNbrplacedispo()
    {
        return this.nbrplacedispo;
    }

    public void setNbrplacedispo(int nbrplacedispo)
    {
        this.nbrplacedispo = nbrplacedispo;
    }

    public boolean isSuivi()
    {
        return this.suivie;
    }

    public void setSuivi(boolean suivi)
    {
        this.suivie = suivi;
    }

    public String getDatepub()
    {
        return this.datepub;
    }

    public void setDatepub(String datepub)
    {
        this.datepub = datepub;
    }

    public Vehicule getVehciule() {
        return vehciule;
    }

    public void setVehciule(Vehicule vehciule) {
        this.vehciule = vehciule;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }
    
    

    @Override
    public String toString() {
        return "Trajet{" + "id=" + id + ", destination=" + destination + ", depart=" + depart + ", datedepart=" + datedepart + ", datepub=" + datepub + ", cout=" + cout + ", description=" + description + ", nbrplacedispo=" + nbrplacedispo + ", suivie=" + suivie + '}';
    }
    

}

File: Web/src/Pidev/GestionTrajetsBundle/Entity/Trajet.php
<?php

namespace Pidev\GestionTrajetsBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Trajet
 *
 * @ORM\Table(name="trajet", indexes={@ORM\Index(name="id_vehicule", columns={"id_vehicule"}), @ORM\Index(name="id_membre", columns={"id_membre"})})
 * @ORM\Entity(repositoryClass="Pidev\GestionTrajetsBundle\Repository\TrajetRepository")
 */
class Trajet
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="destination", type="string", length=30, nullable=false)
     */
    private $destination;

    /**
     * @var string
     *
     * @ORM\Column(name="depart", type="string", length=30, nullable=false)
     */
    private $depart;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateDepart", type="date", nullable=false)
     */
    private $datedepart;


    /**
     * @var \DateTime
     *
     * @ORM\Column(name="datePub", type="date", nullable=false)
     */
    private $datepub;


    /**
     * @var float
     *
     * @ORM\Column(name="cout", type="float", precision=10, scale=0, nullable=false)
     */
    private $cout;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="text", length=65535, nullable=true)
     */
    private $description;

    /**
     * @var integer
     *
     * @ORM\Column(name="nbrPlaceDispo", type="integer", nullable=false)
     */
    private $nbrplacedispo;

    /**
     * @var boolean
     *
     * @ORM\Column(name="suivi", type="boolean", nullable=false)
     */
    private $suivi;

    /**
     * @var \Vehicule
     *
     * @ORM\ManyToOne(targetEntity="Vehicule")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_vehicule", referencedColumnName="id")
     * })
     */
    private $idVehicule;

    /**
     * @var \Membre
     *
     * @ORM\ManyToOne(targetEntity="Membre")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_membre", referencedColumnName="id")
     * })
     */
    private $idMembre;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }
    /**
     * @return string
     */
    public function getDestination()
    {
        return $this->destination;
    }

    /**
     * @param string $destination
     */
    public function setDestination($destination)
    {
        $this->destination = $destination;
    }

    /**
     * @return string
     */
    public function getDepart()
    {
        return $this->depart;
    }

    /**
     * @param string $depart
     */
    public function setDepart($depart)
    {
        $this->depart = $depart;
    }

    /**
     * @return \DateTime
     */
    public function getDatedepart()
    {
        return $this->datedepart;
    }

    /**
     * @param \DateTime $datedepart
     */
    public function setDatedepart($datedepart)
    {
        $this->datedepart = $datedepart;
    }

    /**
     * @return float
     */
    public function getCout()
    {
        return $this->cout;
    }

    /**
     * @param float $cout
     */
    public function setCout($cout)
    {
        $this->cout = $cout;
    }

    /**
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * @param string $description
     */
    public function setDescription($description)
    {
        $this->description = $description;
    }

    /**
     * @return int
     */
    public function getNbrplacedispo()
    {
        return $this->nbrplacedispo;
    }

    /**
     * @param int $nbrplacedispo
     */
    public function setNbrplacedispo($nbrplacedispo)
    {
        $this->nbrplacedispo = $nbrplacedispo;
    }

    /**
     * @return bool
     */
    public function isSuivi()
    {
        return $this->suivi;
    }

    /**
     * @param bool $suivi
     */
    public function setSuivi($suivi)
    {
        $this->suivi = $suivi;
    }

    /**
     * @return \Vehicule
     */
    public function getIdVehicule()
    {
        return $this->idVehicule;
    }

    /**
     * @param \Vehicule $idVehicule
     */
    public function setIdVehicule($idVehicule)
    {
        $this->idVehicule = $idVehicule;
    }

    /**
     * @return \Membre
     */
    public function getIdMembre()
    {
        return $this->idMembre;
    }

    /**
     * @param \Membre $idMembre
     */
    public function setIdMembre($idMembre)
    {
        $this->idMembre = $idMembre;
    }

    /**
     * @return \DateTime
     */
    public function getDatepub()
    {
        return $this->datepub;
    }

    /**
     * @param \DateTime $datepub
     */
    public function setDatepub($datepub)
    {
        $this->datepub = $datepub;
    }



}

